app/src/main/java/com/android/shelfLife/model/user/UserRepository.kt

```diff
@@ -62,16 +53,16 @@ interface UserRepository {
   suspend fun updateSelectedHousehold(selectedHouseholdUID: String)
 
   /** @param userEmails - The set of user emails to get the user IDs for. */
-  fun getUserIds(userEmails: Set<String?>, callback: (Map<String, String>) -> Unit)
+  suspend fun getUserIds(userEmails: Set<String?>): Map<String, String>
```

Comment:
We found out we were not supposed to use callbacks in our repository function. You have correctly made the changes to make our code correct